### PR TITLE
Revise GPT prompt

### PR DESCRIPTION
**Pull request by macnult**
General:

* [✓] Have you followed the guidelines in our [Contributing](https://github.com/ryansurf/cli-surf/blob/main/CONTRIBUTING.md) document?
* [✓] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/ryansurf/cli-surf/pulls) for the same update/change?

### Code:

[✓] Does your submission pass [tests](https://ryansurf.github.io/cli-surf/tests/)?

### Documentation:

Revised the GPT prompt in response to this reported issue: https://github.com/ryansurf/cli-surf/issues/146 
I also added to the assertion to see what's returned if the test fails.

## Summary by Sourcery

Revise the GPT prompt in the test to ensure precise response matching and enhance the assertion to provide detailed feedback on test failures.

Enhancements:
- Improve the assertion in the test to provide more detailed feedback when the test fails.

Tests:
- Revise the GPT prompt in the test to ensure the response strictly matches the expected phrase.

### CODE REVIEW

File: tests/test_gpt.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def test_simple_gpt():
     "gpt works". If anything else is outputted, we can
     assume an error has occured
     """
-    surf_summary = "Please only output: 'gpt works!' "
+    surf_summary = "Please respond with the exact phrase 'gpt works'. Do not include any additional text or context."
     gpt_prompt = "This is for testing purposes"
     gpt_response = gpt.simple_gpt(surf_summary, gpt_prompt)
-    assert "gpt works" in gpt_response
+    assert "gpt works" in gpt_response, f"Expected 'gpt works', but got: {gpt_response}"
```
[the remark this answers is not in the document]
Comment (macnult) [reply tests/test_gpt.py:20]: Changing this resolved the issue.

**Comment by macnult** [comment]
The test still passes/fails inconsistently. When it does fail, it returns the following:
_____________________________________________
assert gpt_response == expected_response, f"Expected '{expected_response}', but got: {gpt_response}"
E       AssertionError: Expected 'gpt works', but got: I'm here to help with any inquiries you may have. How can I assist you today?
E       assert "I'm here to ...st you today?" == 'gpt works'
E
E         - gpt works
E         + I'm here to help with any inquiries you may have. How can I assist you today?

tests\test_gpt.py:22: AssertionError
_____________________________________________

The same thing happens with test_helper.py, although not as frequent. Locally I changed gpt_prompt in test_helper.py which helps during some tests, but isn't consistent to where I can call it resolved.

**Comment by ryansurf** [comment]
> The test still passes/fails inconsistently. When it does fail, it returns the following:
> 
> assert gpt_response == expected_response, f"Expected '{expected_response}', but got: {gpt_response}" E AssertionError: Expected 'gpt works', but got: I'm here to help with any inquiries you may have. How can I assist you today? E assert "I'm here to ...st you today?" == 'gpt works' E E - gpt works E + I'm here to help with any inquiries you may have. How can I assist you today?
> 
> tests\test_gpt.py:22: AssertionError
> 
> The same thing happens with test_helper.py, although not as frequent. Locally I changed gpt_prompt in test_helper.py which helps during some tests, but isn't consistent to where I can call it resolved.



> The test still passes/fails inconsistently. When it does fail, it returns the following:
> 
> assert gpt_response == expected_response, f"Expected '{expected_response}', but got: {gpt_response}" E AssertionError: Expected 'gpt works', but got: I'm here to help with any inquiries you may have. How can I assist you today? E assert "I'm here to ...st you today?" == 'gpt works' E E - gpt works E + I'm here to help with any inquiries you may have. How can I assist you today?
> 
> tests\test_gpt.py:22: AssertionError
> 
> The same thing happens with test_helper.py, although not as frequent. Locally I changed gpt_prompt in test_helper.py which helps during some tests, but isn't consistent to where I can call it resolved.

The gpt is definitely finicky, but your prompt seems to be the better choice. Might have to rethink this going forward but I'll merge your PR as its an improvement!

**Comment by ryansurf** [comment]
@macnult whoops, actually the linter [failed](https://github.com/ryansurf/cli-surf/actions/runs/11241299858/job/31595968782?pr=147). Can you run `make lint` to fix the error?

**Comment by macnult** [comment]
@ryansurf Sure thing! Should be good to go now.

**Comment by ryansurf** [comment]
@all-contributors please add @macnult for code

**Pull request merged**